amd_openvx/openvx/ago/ago_interface.cpp

```diff
@@ -1406,7 +1409,7 @@ vx_status agoVerifyNode(AgoNode * node)
 					}
 					// re-initialize, if updated
 					if (updated) {
-						char scale[256], desc[512];
```

Comment:
any reason to reduce the size?

Reply:
I thought that was from the changes one of you commited. I did a merge with your branch before pushing my changes

Reply:
Changed it to what it was. The rest of the changes are mine

Reply:
LGTM